solr/contrib/ltr/src/java/org/apache/solr/ltr/response/transform/LTRFeatureLoggerTransformerFactory.java

```diff
@@ -187,8 +187,8 @@ public String getName() {
     }
 
     @Override
-    public void setContext(ResultContext context) {
```

Comment:
Hey @m-khl  thanks for reviewing my code! 

> here is the problem: we have a plenty of transformers they are not changed? (assuming they shouldn't be. see below)

In the main abstract class I only deprecated `DocTransformer::setContext`, so they will still compile. I see your point, and I could change so that `prepare` calls also `setContext`, in this way the behaviour of existing transformers will be preserved